client/src/components/History/Content/Dataset/DatasetMiscInfo.vue

```diff
@@ -16,21 +16,19 @@ const knownErrors = [{ regex: sharingErrorRex, modalRef: sharingError }];
 const props = defineProps<Props>();
 
 const fixable = computed(() => {
-    return true;
```

Comment:
Well played John 😆 - thanks for the fix.

Reply:
everything is fixable 😆